setup.bash

```diff
@@ -0,0 +1,2 @@
+
+export PYTHONPATH=$PWD/src:$PYTHONPATH
```

Comment:
If this is specifically so that pytest works I discovered that having an empty file named conftest.py at the root level signals pytest to use it in the path. No need to source.

Reply:
Remove this file and added new conftest.py file.